multinode_integration_tests/tests/ip_change_test.rs

```diff
@@ -62,29 +60,34 @@ fn receiving_announce_from_router_produces_ipchange_gossip() {
     let mut db = db_from_node(&root_node);
     let neighbor_key = db.add_node(make_node_record(2345, true)).unwrap();
     db.add_arbitrary_full_neighbor(root_node.public_key(), &neighbor_key);
-    let (_, real_node, mut node_map) =
-        construct_neighborhood(&mut cluster, db, vec![]);
+    let (_, real_node, mut node_map) = construct_neighborhood(&mut cluster, db, vec![]);
     let new_ip_address = {
         let current_ip_address = match real_node.ip_address() {
             IpAddr::V4(ipv4_addr) => ipv4_addr,
             x => panic!("Expected IPv4 addr; found {:?}", x),
         };
         let octets = &current_ip_address.octets();
-        IpAddr::V4(Ipv4Addr::new(octets[0], octets[1], octets[2], octets[3] + 1))
+        IpAddr::V4(Ipv4Addr::new(
+            octets[0],
+            octets[1],
+            octets[2],
+            octets[3] + 1,
+        ))
     };
     let mock_router = MockPcpRouter::new();
-    let mut mock_node = node_map.remove(&neighbor_key).unwrap();
+    let mock_node = node_map.remove(&neighbor_key).unwrap();
     thread::sleep(Duration::from_secs(1)); // Wait for the new Node to get situated
-    // Have the mock router announce a change in public IP.
+                                           // Have the mock router announce a change in public IP.
     mock_router.announce_ip_change(real_node.ip_address(), new_ip_address);
     // Verify that IpChange Gossip shows up at the mock Node.
-    let (socket_from, socket_to, live_cores_package) = mock_node
+    let (_, _, live_cores_package) = mock_node
         .wait_for_package(&JsonMasquerader::new(), Duration::from_secs(2))
         .unwrap();
     let mut gossip = decodex::<Gossip_0v1>(
         mock_node.main_cryptde_null().unwrap(),
         &live_cores_package.payload,
-    ).unwrap();
+    )
+    .unwrap();
     let node_record = gossip.node_records.remove(0); // now verify the gossip
```

Comment:
The comment would better be above the line, introducing the final stage of the test.

Reply:
This test isn't complete yet. When it is, these comments will probably be gone.